docs/application/web/api/4.0/device_api/tv/index.html

```diff
@@ -176,19 +176,19 @@ <h4 id="TV Controls">TV Controls</h4>
 <td>No</td>
 </tr>
 <tr><td><a href="tizen/tvinfo.html">TV Information</a></td>
-<td>This API allows receiving key events generated when the user presses a key of an Input Device (for example a TV remote control).</td>
+<td>This API provides interfaces and methods for getting information about TV settings.</td>
```

Comment:
This API provides interfaces and methods to get information about the TV settings.